codalab/rest/bundles.py

```diff
@@ -492,7 +590,14 @@ def _fetch_bundle_stores():
     - `storage_format`: the format in which storage is being stored (UNCOMPRESSED, COMPRESSED_V1, etc)
     - `url`: a self-referential URL that points to the bundle store.
     """
-    bundle_stores = local.model.get_bundle_stores(request.user.user_id)
+    bundle_store_name = query_get_type(str, 'name', None)
+    if bundle_store_name is not None:
+        # this function only fetch one record
```

Comment:
What is this change for? Should we update the docstring as well?

Reply:
This is a improvement of this function: support query a bundle store by bundle store name. We don't need to update the docs, cause the description of the endpoint has already have the parameter `name`, but we did not support using the `name` field before.

Reply:
Wait, but the description of the endpoint doesn't already have the parameter `name`. So should we update the docs?